pkg/execution/driver/httpdriver/dialer.go

```diff
@@ -0,0 +1,128 @@
+package httpdriver
+
+import (
+	"context"
+	"fmt"
+	"net"
+)
+
+var privateIPBlocks []*net.IPNet
+var nat64blocks []*net.IPNet
+
+func init() {
+	for _, cidr := range []string{
+		"127.0.0.0/8",    // IPv4 loopback
```

Comment:
Why not use [`IsLoopback`](https://pkg.go.dev/net#IP.IsLoopback) instead?

Reply:
we can, but this is now one loop for all banned IPs.

Reply:
We also do call that in `isPrivateIP` :)